modules/tinymce/src/themes/silver/test/ts/browser/editor/SilverFixedToolbarContainerPriorityTest.ts

```diff
@@ -0,0 +1,125 @@
+﻿import { ApproxStructure, Assertions } from '@ephox/agar';
+import { after, before, describe, it } from '@ephox/bedrock-client';
+import { TinyHooks } from '@ephox/mcagar';
+import { Insert, Remove, SugarBody, SugarElement } from '@ephox/sugar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('browser.tinymce.themes.silver.editor.SilverFixedToolbarContainerTest', () => {
```

Comment:
```suggestion
describe('browser.tinymce.themes.silver.editor.SilverFixedToolbarContainerPriorityTest', () => {
```

Reply:
Woops. Yeah I definitely didn't copy this from `SilverFixedToolbarContainerTest` 🙄